packages/orbit-components/src/FormLabel/index.tsx

```diff
@@ -1,95 +1,59 @@
-import mq from "../utils/mediaQuery";
 import type { Props } from "./types";
 
-const StyledAsterisk = styled.span<{ filled?: boolean }>`
-  ${({ theme, filled }) => css`
-    font-weight: ${theme.orbit.fontWeightBold};
-    color: ${!filled ? theme.orbit.colorTextError : theme.orbit.colorFormLabelFilled};
-    font-size: ${theme.orbit.fontSizeFormLabel};
-    vertical-align: top;
-  `}
-`;
-
-StyledAsterisk.defaultProps = {
-  theme: defaultTheme,
-};
-
-const StyledInputErrorIcWrapper = styled.span`
-  ${({ theme }) => css`
-    margin: ${rtlSpacing(`0 ${theme.orbit.spaceXXSmall} 0 0`)};
-    display: inline-flex;
-    align-items: center;
-  `};
-`;
-
-StyledInputErrorIcWrapper.defaultProps = {
-  theme: defaultTheme,
-};
-
-const FormLabel = styled(
-  ({
-    className,
-    children,
-    required,
-    filled,
-    dataTest,
-    id,
-    error,
-    help,
-    onMouseEnter,
-    onMouseLeave,
-    iconRef,
-    inlineLabel,
-    labelRef,
-  }: Props) => (
-    <span
-      className={className}
-      data-test={dataTest}
-      id={id}
-      ref={labelRef}
-      onMouseEnter={onMouseEnter}
-      onMouseLeave={onMouseLeave}
-    >
-      {!inlineLabel && (error || help) && (
-        <StyledInputErrorIcWrapper ref={iconRef}>
-          {error && <AlertCircle ariaLabel="error" color="critical" size="small" />}
-          {!error && help && <InformationCircle ariaLabel="help" color="info" size="small" />}
-        </StyledInputErrorIcWrapper>
-      )}
-
-      {required && !error && (
-        <StyledAsterisk aria-hidden="true" filled={filled}>
-          *
-        </StyledAsterisk>
-      )}
-      <span>{children}</span>
-    </span>
-  ),
-)`
-  ${({ theme, filled, disabled }) => css`
-    display: inline-flex;
-    font-family: ${theme.orbit.fontFamily};
-    font-size: ${theme.orbit.fontSizeFormLabel};
-    font-weight: ${theme.orbit.fontWeightMedium};
-    color: ${!filled || disabled ? theme.orbit.colorFormLabel : theme.orbit.colorFormLabelFilled};
-    line-height: ${theme.orbit.lineHeightTextNormal};
-    margin-bottom: ${theme.orbit.spaceXXSmall};
-    margin-${left}: ${theme.orbit.spaceXXSmall};
-    ${mq.desktop(css`
-      margin-${left}: 0;
-    `)}
-  `}
-`;
-
-FormLabel.defaultProps = {
-  theme: defaultTheme,
-};
+const FormLabel = ({
+  className,
+  children,
+  required,
+  filled,
+  dataTest,
+  id,
+  error,
+  help,
+  onMouseEnter,
+  onMouseLeave,
+  iconRef,
+  inlineLabel,
+  labelRef,
+  disabled,
+}: Props) => (
+  <span
+    className={cx(
+      className,
+      "orbit-form-label",
+      "font-base text-normal mb-xxs ms-xxs de:ms-0 inline-flex font-medium leading-normal",
+      !filled || disabled ? "text-ink-dark" : "text-ink-normal",
+    )}
+    data-test={dataTest}
+    id={id}
+    ref={labelRef}
+    onMouseEnter={onMouseEnter}
+    onMouseLeave={onMouseLeave}
+  >
+    {!inlineLabel && (error || help) && (
+      <span className="me-xxs m-0 inline-flex items-center" ref={iconRef}>
```

Comment:
```suggestion
      <span className="me-xxs inline-flex items-center" ref={iconRef}>
```

I think no need for having explicitly set margins with `0`